mapswipe_workers/mapswipe_workers/utils/slack_helper.py

```diff
@@ -27,14 +23,58 @@ def send_slack_message(message_type: str, project_name: str, project_id: str = N
             + "Make sure to activate the project using the manager dashboard.\n"
             + "Happy Swiping. :)"
         )
+        slack_client.chat_postMessage(channel=SLACK_CHANNEL, text=message)
     elif message_type == "fail":
```

Comment:
You could use an [enum](https://docs.python.org/3/library/enum.html) for the `message_type`, it's a bit more structured than using `str`, and you get better type checking and syntax completion, which in turn prevents typos, etc...
```python
from enum import Enum
class MessageType(Enum):
    FAIL = 1
    NOTIFICATION_90 = 2
    ...

Then you can check with `if message_type == MessageType.FAIL`, etc...
You'll need to import the class everywhere you use it, but it's probably a good way to define these centrally.